package.json

```diff
@@ -25,7 +25,8 @@
     "test": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^5.1.7"
+    "ember-cli-babel": "^5.1.7",
+    "eslint-plugin-ember-suave": "^1.0.0"
```

Comment:
Ooops. I think that it's because `yarn add` adds them there, and I'm used to `npm --save-dev` doing the opposite